usaspending_api/broker/management/commands/fabs_nightly_loader.py

```diff
@@ -101,7 +101,7 @@ def get_fabs_records_to_delete(submission_ids, afa_ids, start_datetime, end_date
     sql = """
         select  distinct upper(afa_generated_unique)
         from    published_award_financial_assistance p
-        where   correction_delete_indicatr = 'D' and
+        where   UPPER(correction_delete_indicatr) = 'D' and
```

Comment:
Oh god.  That's my SQL.  Were those 10 records the cause of the discrepancy?

Reply:
The discrepancies were in those 10 records so I think this was the culprit. Still caught it early enough to not be a major issue 👍